modules/network-observability-netobserv-cli-reference.adoc

```diff
@@ -1,13 +1,14 @@
 // Automatically generated by './scripts/generate-doc.sh'. Do not edit, or make the NETOBSERV team aware of the editions.
 :_mod-docs-content-type: REFERENCE
+
 [id="network-observability-netobserv-cli-reference_{context}"]
 = Network Observability CLI usage
 
-You can use the Network Observability CLI (`oc netobserv`) to pass command line arguments to capture flow data and packet data for further analysis, enable Network Observability Operator features, or pass configuration options to the eBPF agent and `flowlogs-pipeline`.
+You can use the Network Observability CLI (`oc netobserv`) to pass command line arguments to capture flows data and packets data for further analysis, enable Network Observability Operator features or pass configuration options to the eBPF agent and `flowlogs-pipeline`.
```

Comment:
Would it be worth to mention metrics here since we introduced these ?

I just realized that

Reply:
Yes that's true we should add metrics to this intro

Reply:
I can add it here; I know you already merged your upstream reference info.

Reply:
I'll update our side according to your changes :wink: 
Thanks !

Reply:
changed this just to add "metrics" in the intro, @jpinsonneau